### PR TITLE
fix: replaces travis with gh actions for v33

### PR DESCRIPTION
**Pull request by paschalidi**

### CODE REVIEW

File: .github/workflows/verify-app.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        with:
+          node-version: 12.x
+
+      - uses: actions/download-artifact@v2
+        with:
+          name: app-build
+
+      - uses: dhis2/deploy-build@master
+        with:
+          build-dir: build/app
```
Comment (varl) [inline .github/workflows/verify-app.yml:82]: In v33 I think this needs to be `build-dir: build` because v33 is before the app-platform port.

Comment (varl) [reply .github/workflows/verify-app.yml:82]: Compare:

https://github.com/d2-ci/capture-app/tree/master
https://github.com/d2-ci/capture-app/tree/v35

and

https://github.com/d2-ci/capture-app/tree/cp/v33/ci/replace-travis

**Review by varl — CHANGES_REQUESTED** [review]

File: .github/workflows/verify-app.yml (hunk 1 above)
Comment (paschalidi) [reply .github/workflows/verify-app.yml:82]: 👍  doing so

Comment (varl) [reply .github/workflows/verify-app.yml]: Nice, thanks.

**Review by varl — APPROVED** [review]
LGTM :shipit:

**Review by JoakimSM — APPROVED** [review]

**Pull request merged**